astropy/cosmology/flrw/tests/test_base.py

```diff
@@ -766,14 +773,12 @@ def test_clone_change_param(self, cosmo):
         assert c.__class__ == cosmo.__class__
         assert c.name == cosmo.name + " (modified)"
         assert c.H0.value == 100
-        for n in set(cosmo.__parameters__) - {"H0"}:
-            v = getattr(c, n)
+        for n, v in filter_keys_from_items(c.parameters.items(), {"H0"}):
```

Comment:
If we are calling a helper function anyways we might as well make full use of that helper by having it call `.items()` for us. And there is no good reason to put `"H0"` in a set here. I'm only pointing it out here, but analogous changes should be made throughout the pull request.

Reply:
✅

Reply:
Done. ``H0`` is now in a tuple.